### ISSUE TITLE
new terms for normal transcription factor activity

### ISSUE THREAD

**Issue by fypoadmin** [pombase/fypo#1199]
I am trying to capture some experiments which show that the atf1 transcription factor can be regulated independently of the Sty1 MAPK pathway.
The authors delete various  pathways genes and  use a new, more sensitive assay show transcription is still present in response to specific stresses

so far so good.

I thought I needed
x transcription factor activity in response to heat stress"
x transcription factor activity in response to oxidative stress
x transcription factor activity in response to heavy metal stress
x transcription factor activity in response to low glucose 

where a TF is defined specifically as a sequence specific DNA binding TF  as having BOTH DNA binding AND regulation of transcription)

?x is the problem.

The expression is _increased_ compared to wild type, but
i) increase is _normal_ under these stresses
and 
ii) the increase here is _reduced_  compared to _normal under these conditions_

so how do I capture this?
Basically to  demonstrate that the TF is still active independently of sty1 pathway (albeit not fully)

Then...
I was going to say "assayed using atf1"
but  a little further on they do another exp
using rT PCR of specific atf1 regulated gene

so this would be the same term with  assayed using hsp9
but how can I capture both the TF being referred to by the term, and the target gene unambigously?

Original comment by: ValWood
